### ISSUE TITLE
Crash after upgrade to v1.1.2 - alchemistry issue?

### ISSUE THREAD

**Issue by JakeMoe** [AllTheMods/All-the-mods-7-Sky#301]
After upgrading to v1.1.2, I was unable to start my server; it would get past loading spawn chunks and then shortly crash afterwards with what appears to be an index out of range error:

```
java.lang.RuntimeException: Slot 11 not in valid range - [0,10)
        at net.minecraftforge.items.ItemStackHandler.validateSlotIndex(ItemStackHandler.java:207) ~[forge-1.18.2-40.1.80-universal.jar%23265!/:?] {re:classloading}
        at net.minecraftforge.items.ItemStackHandler.getStackInSlot(ItemStackHandler.java:59) ~[forge-1.18.2-40.1.80-universal.jar%23265!/:?] {re:classloading}
        at com.smashingmods.alchemistry.api.storage.AutomationSlotHandler.getStackInSlot(AutomationSlotHandler.java:22) ~[alchemistry-1.18.2-2.1.0.jar%2344!/:1.18.2-2.1.0] {re:classloading}
        at com.smashingmods.alchemistry.api.blockentity.processing.AbstractInventoryBlockEntity$2.extractItem(AbstractInventoryBlockEntity.java:72) ~[alchemistry-1.18.2-2.1.0.jar%2344!/:1.18.2-2.1.0] {re:classloading}
```
[crash-2022-10-21_08.53.53-server.txt](https://github.com/AllTheMods/All-the-mods-7-Sky/files/9837564/crash-2022-10-21_08.53.53-server.txt)

Crash log attached.  Reverting alchemistry on the server from 2.1.0 back to the 2.0.9 from server version 1.1.1 allowed the server to start and me to join again.

**Comment by delta2k** [comment]
had the same crash message. for sp. so did the same solution as JakeMoe for sp only and the world loaded fine.

**Comment by jeremiahwinsley** [comment]
Tracked here: https://github.com/SmashingMods/Alchemistry/issues/254

**Comment by JakeMoe** [comment]
Fixed by picking up the Dissolver on the old v2.0.9 version, shutting down the server and client, upgrading both to the new v2.1.2 released to fix this, and then starting up the world again.  And I was able to place the Dissolver back down again without issue after it started up.
